Add docs for non-html route encoding option (#1434)
withastro/docs: src/pages/en/core-concepts/astro-pages.md

};
```

> If you're using SSG (static site generation) and you need to customize the encoding of the resulting file, you can return the `encoding` option in addition to the `body`. It can be any valid [`BufferEncoding`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bdd02508ddb5eebcf701fdb8ffd6e84eabf47885/types/node/buffer.d.ts#L169) accepted by node.js' `fs.writeFile` method. For example, to produce a binary png image using SSG:

Reviewer: ```suggestion
You can optionally return an `encoding` option in static builds. It can be any valid [`BufferEncoding`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/bdd02508ddb5eebcf701fdb8ffd6e84eabf47885/types/node/buffer.d.ts#L169) accepted by node.js' `fs.writeFile` method. For example, to produce a binary png image using SSG:
```
How does this read? Trying to make it a little more concise.

> ```ts title="src/pages/image.png.ts"

Reviewer: ```suggestion
```ts title="src/pages/image.png.ts" {7}
```
Highlight the line `encoding: 'binary',`
